src/Fizzics/skinImage.js

```diff
@@ -14,6 +14,10 @@ var FizzicsSkinImage = GObject.registerClass({
             'pixels', 'pixels', 'pixels',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT,
             0, 64, 64),
+        path: GObject.ParamSpec.string(
+            'path', 'path', 'path',
+            GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT,
+            '/com/endlessm/HackToolbox/Fizzics/skins'),
```

Comment:
This is not really a path, it's a resource path. Perhaps consider renaming this property.